docs/content/dagster-cloud/managing-deployments/setting-up-alerts.mdx

```diff
@@ -56,14 +55,20 @@ Currently, Slack and email notifications are supported.
 
 4. Click **+ Create alert policy**.
 
-5. From the **Alert Policy type** drop-down, select **Run alert**, **Schedule/Sensor alert**, **Code location error alert**, or **Agent downtime alert**.
+5. From the **Alert Policy type** drop-down, select the type of alert to create.
 
 6. In the **Create alert policy** window, fill in the following:
 
    - **Alert policy name** - Enter a name for the alert policy. For example, `slack_urgent_failure`
    - **Description** - Enter a description for the alert policy
 
-   For run-based alerts, fill out these additional options:
+   For asset-based alerts, fill out these additional options:
+
+   - **Asset group** - Select the asset group to monitor. You will have the option to select from all asset groups in the deployment.
+   - **Asset key** - Select the asset key to monitor. You will have the option to select from all asset keys in the deployment. (Note: If you select an asset group, you will not be able to select an asset key.)
```

Comment:
```suggestion
   - **Asset key** - Select the asset key to monitor. You will have the option to select from all asset keys in the deployment. **Note:** If you select an asset group, you will not be able to select an asset key.
```

Also a nit - this is how I've been formatting/asking others to format asides like this